src/components/VScrollButton.vue

```diff
@@ -2,7 +2,7 @@
   <button
     :aria-label="$t('browse-page.aria.scroll')"
     type="button"
-    class="scroll text-white bg-trans-blue hover:bg-trans-blue-action transition-all duration-100 ease-linear fixed bottom-4 w-14 h-14 hover:shadow-md rounded-full text-center"
+    class="scroll text-white bg-trans-pink hover:bg-trans-pink-action transition-all duration-100 ease-linear fixed bottom-4 w-14 h-14 hover:shadow-md rounded-full text-center"
```

Comment:
Could  you change the colors:
~`bg-trans-pink`~: `bg-pink`
~`hover:bg-trans-pink-action`~: `bg-dark-pink`

Reply:
Thank you for your PR, @kb-0311 ! We are using Tailwind, and only some pre-configured colors are allowed in the class names. Here are the definitions of the pink that the button should use:
https://github.com/WordPress/openverse-frontend/blob/e23af1ac77da7f74cfa808fb0ef952666a6c7e2a/tailwind.config.js#L27-L31
There are no such colors as `bg-trans-blue`, that's why you are seeing the errors.

Reply:
ohhh got it thanks also I made the changes you requested.

Reply:
@obulat actually I am pretty sure I added the dark class instead of the trans class however still giving some E2E error.

Reply:
Great! Your changes are correct, and the e2e failure is not related to them. Let me look into that.

Reply:
ok thanks for your feedback hoping to contribute more to the project in the future

Reply:
@kb-0311 , I think #1041 should fix the e2e failure, and once it's reviewed and merged, your PR should also pass.
Thank you for your contribution, and welcome to Openverse :)